vnext/Microsoft.ReactNative/Fabric/Composition/CompositionDynamicAutomationProvider.cpp

```diff
@@ -381,6 +381,8 @@ HRESULT __stdcall CompositionDynamicAutomationProvider::Invoke() {
   if (spProviderSimple != nullptr) {
     UiaRaiseAutomationEvent(spProviderSimple.get(), UIA_Invoke_InvokedEventId);
   }
+  DispatchAccessibilityAction(m_view, "invoke");
```

Comment:
Is there a difference between these two?

Reply:
I believe the action should be the same. "activate" is just the action name that RN is using and "invoke" is the Windows name for the same action. I figured we should just support both names, so the user can either use the windows action name or the cross-plat action name.